translate/cloud-client/translate_v3_translate_text_with_model.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START translate_v3_translate_text_with_model]
+
+from google.cloud import translate
+
+
+def translate_text_with_model(
+    text="YOUR_TEXT_TO_TRANSLATE",
```

Comment:
It'd be nice to inline this as well, but I know testing might get broken by that.

Reply:
Is it only for this one or all translate_Text?

Reply:
Ideally all, where possible.